distributed/tests/test_semaphore.py

```diff
@@ -391,9 +392,12 @@ def observe_state(sem):
             x_locked = client.get_metadata(0) == "locked"
             y_locked = client.get_metadata(1) == "locked"
 
+        # Once both are locked we should give the refresh time to notify the scheduler
+        # This parameter should be larger than ``lease-validation-interval``
+        slowidentity(delay=0.15)
```

Comment:
lease-validation-interval is 10ms, do you mean `0.015`?

Reply:
well, anything larger should do